content/README.md

```diff
@@ -0,0 +1,124 @@
+# CWL Website 2.0
+
+The in-progress redesign of the CWL Website
+
+The preview build of the site is currently found at [https://deploy-preview-75--cwl-website.netlify.app/](https://deploy-preview-75--cwl-website.netlify.app/)
+
+## To-Do
+
+### Mobile
+
+* Mobile nav should close when you tap outside of it
+* Ensure "Back to Top" button shows up for long pages
+* Users Gallery Submenus should be drop downs
+
+### Broken Links
+
+Users Gallery - Jekyll is inserting a blank link between US National Institutes of Health (NIH) & 
+RAPT. The code renders like this:
+
+```html
+<div class="adopter-entry">
+  <div class="adopter-row-top">
+    <a href="" class="adopter-link"></a>
+  </div>
+</div>
+```
+
+### WebAIM Accessibility
+
+#### Video Player
+
+* The "broken ARIA menu" seems to be an error on the WebAIM tool's part...
+  - It says that it has to have at least one item with `role="menuitem"` - which is satistfied...
+  - Test with screenreader or emulator if possible
+
+```html
+<div id="plyr-settings-9859-home">
+  <div role="menu">
+    <button data-plyr="settings" type="button" class="plyr__control plyr__control--forward" role="menuitem" aria-haspopup="true">
+      <span>Captions<span class="plyr__menu__value">Disabled</span>
+      </span>
+    </button>
+    <button data-plyr="settings" type="button" class="plyr__control plyr__control--forward" role="menuitem" aria-haspopup="true" hidden="">
+      <span>Quality<span class="plyr__menu__value">undefined</span>
+      </span>
+    </button>
+    <button data-plyr="settings" type="button" class="plyr__control plyr__control--forward" role="menuitem" aria-haspopup="true">
+      <span>Speed<span class="plyr__menu__value">Normal</span>
+      </span>
+    </button>
+  </div>
+</div>
+```
+
+These other errors come up, but are probably not significant, since the icons are provided:
+
+* skipped heading level
+* Small text
+
+### Cleanup & Deployment
+
+* Ensure hypothes.is is off for live site
```

Comment:
Yep - it's only enabled for internal feedback on the staging site. Can I ask where/when in the navigation the screen reader announces it? Perhaps it would be worth either hiding or re-ordering for screen readers. 

@swzCuroverse @mr-c any thoughts?